src/misc.cpp

```diff
@@ -77,3 +78,55 @@ std::ostream& operator<<(std::ostream& dest, __int128_t value)
 }
 
 } // namespace std
+
+namespace quadiron {
+
+std::ostream& hex_dump(
```

Comment:
Yes I took this code from somewhere as I really needed to dump buffers. I think it makes sense to keep this routine here because it is so useful for debugging.

The liberasure version may eventually disappear.

Reply:
Then let's open an issue about it, otherwise it will be forgotten and never done

Reply:
created issue #270